jirasync.py

```diff
@@ -314,7 +315,6 @@ def start_syncing_team(config, interval):
                 teams[team]['github_token']
             )
 
-            created_issue_list = git_hub_plugin.get_github_issues_created_list()
```

Comment:
@rochacbruno Here was the code trying to fetch issue list even it was unassigned.